ui/app/components/secure-variable-form.js

```diff
@@ -0,0 +1,20 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+
+export default class SecureVariableFormComponent extends Component {
+  keyValues = [{ key: '', value: '' }];
+
+  @action appendRow() {
+    this.keyValues.pushObject({
+      key: '',
+      value: '',
+    });
+  }
+
+  @action
+  saveNewVariable(e) {
+    e.preventDefault();
+    const { path, keyValues } = this;
+    this.args.save({ path, keyValues });
```

Comment:
`question` `thought` `non-blocking`: This is eventually going to be an asynchronous action, correct?

We have a `Trigger` component that you might want to check out when that time comes.